server/views/commit/index.jade

```diff
@@ -3,7 +3,7 @@ block content
     h2.text-center Commit to yourself. Commit to a nonprofit.
     .row
         .col-xs-12.col-sm-6.col-sm-offset-3
-          p Give yourself external motivation and help nonprofits right away. Pledge a monthly donation to a nonprofit until you’ve earned either your Front End or Full Stack Development Certification.
+          p You can give yourself external motivation, and immediately start helping nonprofits. You can do this by pledging a monthly donation to a nonprofit until you’ve earned one of our certifications. This pledge is completely optional. This pledge is entirely between you and the nonprofit, and no money goes to Free Code Camp. You can change your commitment or stop it at any time.
```

Comment:
~~This pledge is completely optional. This pledge is entirely between you and the nonprofit, and no money goes to Free Code Camp.~~ This pledge is completely optional, entirely between you and the nonprofit, and no money goes to Free Code Camp.